public/app/plugins/panel/geomap/layers/data/photosLayer.tsx

```diff
@@ -124,7 +117,7 @@ export const photosLayer: MapLayerRegistryItem<PhotoConfig> = {
           radius: config.radius,
           crop: false,
           kind: config.kind,
-          shadow: config.shadow,
+          shadow: config.shadow ? 2 : 0,
```

Comment:
@grafana/dataviz-squad I would appreciate your eyes on the changes in this file along with taking it for a test locally to confirm it works as expected. I have very little domain knowledge here I'm afraid.